cmd/server/assets/home.html

```diff
@@ -342,7 +342,7 @@ <h1>Create verification code</h1>
 
     function getCode(data) {
       $.ajax({
-        url: '/home/issue',
+        url: '/code/issue',
```

Comment:
I'm not super excited about this change - if someone is logged into the system during an upgrade, their live page will stop working.

Reply:
We can keep around the /home/issue path for a release.